tests/integration/testing_timing/testing_timing_test.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+from typing import Set
+
+import pytest
+from protostar.commands.test.testing_summary import TestingSummary
+from protostar.utils.log_color_provider import LogColorProvider
+from tests.integration.conftest import RunCairoTestRunnerFixture
+
+
+def assert_exec_times_in_desc_order(logs: str) -> None:
+    raw_exec_times = [row.split()[3] for row in logs.split("\n")]
+    exec_times = [
+        float(exec_time.lstrip("(time=").rstrip("s)")) for exec_time in raw_exec_times
+    ]  # "(time={NUMBER}s)" -> {NUMBER}
+    assert exec_times == sorted(exec_times, reverse=True)
```

Comment:
I think this assertion may be faulty if there are two tests with different names but exactly the same timings. This assertion may not check that tests are sorted by name as expected.

Why not simply asserting that the logged output matches a regular expression?